core/lpms.go

```diff
@@ -131,6 +133,19 @@ func (l *LPMS) HandleHLSPlay(
 	l.vidPlayer.HandleHLSPlay(getMasterPlaylist, getMediaPlaylist, getSegment)
 }
 
+func (l *LPMS) RTMPToHLS(s *segmenter.FFMpegVideoSegmenter, ctx context.Context, cleanup bool) error{
+	var err error
+	for i:=0; i < RETRY_COUNT; i++ {
```

Comment:
Ideally we'd continue until the RTMP stream is terminates but this isn't so easily done since the `RtmpVideoStream` interface doesn't expose the information we need. Fixing that is probably outside the scope of this bounty, so this is probably OK for now.